### PR TITLE
Add kafka prefix to kakfa topic names

### PR DESCRIPTION
**Pull request by SheltonZSL**
Add new configuration `kafka_namespace` which will be used a prefix of the OAP-created Kafka topics.

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue url. Closes: https://github.com/apache/skywalking/issues/10317
- [x] Update the [`CHANGELOG.md`](https://github.com/apache/skywalking-python/blob/master/CHANGELOG.md).

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
   - Add Greenlet profiler (#246)
   - Add test and support for Python Slim base images (#249)
   - Add support for the tags of Virtual Cache for Redis (#263)
+  - Add a new configuration `kafka_namespace` to prefix the kafka topic names (#276)
```
Comment (Superskyyy) [inline CHANGELOG.md:20]:
```suggestion
  - Add a new configuration `kafka_namespace` to prefix the kafka topic names (#277)
```

**Review by Superskyyy — CHANGES_REQUESTED** [review]
One nit, then it's good

**Comment by SheltonZSL** [comment]
OK, I got it.

**Review by Superskyyy — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Pull request merged**